src/taco-dependency-installer/installers/androidSdkInstaller.ts

```diff
@@ -287,7 +287,7 @@ class AndroidSdkInstaller extends InstallerBase {
 
     private postInstallDefault(): Q.Promise<any> {
         var self = this;
-        return this.installAndroidPackages()
+        return this.installANDROID_PACKAGES()
```

Comment:
If you change the method name back to "installAndroidPackages" don't forget to change this line back as well